spec/symbol_spec.rb

```diff
@@ -36,13 +36,13 @@
   end
 
   context 'with an array of Ruby Strings as an argument' do
-    let(:args) { [%w[x y]] } # it would be one Array argument on *args
+    let(:args) { [%w(x y)] } # it would be one Array argument on *args
 
     it_behaves_like 'two symbols'
   end
 
   context 'with a splatted argument' do
-    let(:args) { %w[x y] } # it would be several separate strings on *args
+    let(:args) { %w(x y) } # it would be several separate strings on *args
```

Comment:
Again, matter of personal style, but it is easy to redefine in `.rubocop.yml` what is appropriate limitation characters for `%w` arrays. And for me, `%w[]` is much more readable.